config/routes.rb

```diff
@@ -163,6 +163,10 @@
       scope module: :forum do
         resources :forums do
           resources :topics do
+            resources :posts, except: [:new, :show, :index] do
+              get 'reply', on: :member
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.